doc/vim-go.txt

```diff
@@ -478,6 +503,20 @@ documentation in the |go-commands| section. Available <Plug> keys are:
 Calls `go run` for the current file
 
 
+The next three options are neovim only
```

Comment:
Just change like you did for the others by prepending the sentence: `This option is neovim only`